Providers/nxOMSAutomationWorker/automationworker/worker/jrdsclient.py

```diff
@@ -22,7 +25,19 @@ def __init__(self, http_client):
     @staticmethod
     def issue_request(request_function):
         """Invokes request_funtion."""
-        response = request_function
+        response = request_function()
+        tries = 1
+        max_tries = 5
+        while response.status_code in transient_status_codes:
+            if tries > max_tries:
```

Comment:
Please consider logging the fact that a retry had to happen and why to the worker log (guaranteed log) and as an event all the way to Kusto (best effort).

Reply:
Trying to log to Kusto with JRDS logger would cause a recursion. Added changes for local logging.